test/sortedvector.jl

```diff
@@ -5,6 +5,11 @@ A = AxisArray(reshape(1:16, 8, 2), v, [:a, :b])
 @test A[ClosedInterval(8.,12.), :] == A[2:5, :]
 @test A[1. .. 1., :] == A[1:1, :]
 @test A[10. .. 10., :] == A[2:3, :]
+@test size(v) == (8,)
+@test size(v, 1) == 8
+@test size(v, 2) == 1
+# test StepRange indexing
+@test v[1:2:8].data == [1.0; 10.0; 12.0; 14.0]
```

Comment:
We should do something instead of referencing the internal fields. I'm guessing you did this because `v[1:2:8] == [1.0; 10.0; 12.0; 14.0]` didn't work? I think it should be made to work if it doesn't right now.

Reply:
Actually, I was wrong, it does work. Look what happens when you assume, kids!